frontend/app/work_packages/controllers/work-package-details-controller.js

```diff
@@ -71,6 +72,7 @@ module.exports = function($scope,
       });
   }
   $scope.refreshWorkPackage = refreshWorkPackage; // expose to child controllers
+  $rootScope.refreshWorkPackage = refreshWorkPackage; // expose to child controllers
```

Comment:
Why are you exposing the function when the event is already exposed? (`workPackageRefreshRequired`). It does also take a callback so I see no reason to expose the function itself here.